openapi/components/schemas/TransactionAgreement.yaml

```diff
@@ -1,29 +1,13 @@
 description: ''
```

Comment:
While you're here, let's try better writing a better description for this object.

Reply:
I added something, but I'm not sure how good that one is.